nats/aio/subscription.py

```diff
@@ -302,7 +304,8 @@ async def _wait_for_msgs(self, error_cb) -> None:
                     self._pending_queue.task_done()
 
                 # Apply auto unsubscribe checks after having processed last msg.
-                if self._max_msgs > 0 and self._received >= self._max_msgs and self._pending_queue.empty:
+                if self._max_msgs > 0 and self._received >= self._max_msgs and self._pending_queue.empty(
+                ):
```

Comment:
I know it's not the right place to discuss this, but yapf is a very bad autoformatter. What the hell is this? How on the Earth this is readable?

Reply:
sorry for the late reply, I'm ok with removing yapf if you want or use a replacement

Reply:
I was just complaining out loud, I didn't expect a reply. But I'm glad you are open to alternatives. I maintain the [awesome-python-code-formatters](https://github.com/life4/awesome-python-code-formatters) list, so I'd love to bring some good alternative code formatters into the project :) I'll do that when we get Taskfile merged, so I can provide you with a good automation for that.